src/components/modal-container.tsx

```diff
@@ -26,8 +26,8 @@ import document from 'global/document';
 
 import {EXPORT_DATA_TYPE_OPTIONS, EXPORT_MAP_FORMATS} from '@kepler.gl/constants';
 import ModalDialogFactory from './modals/modal-dialog';
-import {exportJson, exportHtml, exportData, exportImage, exportMap} from 'utils/export-utils';
-import {isValidMapInfo} from 'utils/map-info-utils';
+import {exportHtml, isValidMapInfo, exportMap, exportJson} from '@kepler.gl/utils';
+import {exportData} from 'reducers/layer-utils/export-utils';
```

Comment:
`exportData` is not used by any reducer function, it's only used by component, can we move it to `@kepler.gl/utils`?